pages/models.py

```diff
@@ -58,6 +59,27 @@ class BasePage(Page):
             ImageChooserPanel('social_image')
         ]
 
+@register_snippet
+@python_2_unicode_compatible  # provide equivalent __unicode__ and __str__ methods on Python 2
```

Comment:
Not really sure what this does - copied from http://docs.wagtail.io/en/v1.10.1/topics/snippets.html#snippet-models